frontend/packages/core/src/NPS/stories/banner.stories.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import { MemoryRouter } from "react-router";
+import type { Meta } from "@storybook/react";
+
+import Header from "../../AppLayout/header";
+import { ApplicationContext } from "../../Contexts";
+import type { HeaderItems, TriggeredHeaderData } from "../../Contexts/app-context";
+import type { BannerFeedbackProps } from "../banner";
+import { Banner } from "../banner";
+
+export default {
+  title: "Core/NPS/Banner",
+  component: Banner,
+} as Meta;
+
+const Template = ({ ...props }: BannerFeedbackProps) => {
+  const [triggeredHeaderData, setTriggeredHeaderData] = React.useState<TriggeredHeaderData>({});
+  return (
+    <ApplicationContext.Provider
+      // eslint-disable-next-line react/jsx-no-constructed-context-values
+      value={{
+        workflows: [
+          {
+            developer: { name: "Lyft", contactUrl: "mailto:hello@clutch.sh" },
+            displayName: "EC2",
+            group: "AWS",
+            path: "ec2",
+            routes: [
+              {
+                // eslint-disable-next-line react/no-unstable-nested-components
+                component: () => <div>Terminate Instance</div>,
+                componentProps: { resolverType: "clutch.aws.ec2.v1.Instance" },
+                description: "Terminate an EC2 instance.",
+                displayName: "Terminate Instance",
+                path: "instance/terminate",
+                requiredConfigProps: ["resolverType"],
+                trending: true,
+              },
+              {
+                // eslint-disable-next-line react/no-unstable-nested-components
+                component: () => <div>Resize ASG</div>,
+                componentProps: { resolverType: "clutch.aws.ec2.v1.AutoscalingGroup" },
+                description: "Resize an autoscaling group.",
+                displayName: "Resize Autoscaling Group",
+                path: "asg/resize",
+                requiredConfigProps: ["resolverType"],
+              },
+            ],
+          },
+        ],
+        triggerHeaderItem: (item: HeaderItems, open: boolean, data: unknown) =>
```

Comment:
All of the properties are handled via the docs tab in storybook, trying to get away from adding an individual story if it can be handled already

![Kapture 2023-01-09 at 11 13 59](https://user-images.githubusercontent.com/8338893/211389248-665ceddd-a18d-4e29-bcaf-52be3951d4aa.gif)